sdk/database/helper/connutil/sql.go

```diff
@@ -29,8 +50,15 @@ type SQLConnectionProducer struct {
 	MaxConnectionLifetimeRaw interface{} `json:"max_connection_lifetime" mapstructure:"max_connection_lifetime" structs:"max_connection_lifetime"`
 	Username                 string      `json:"username" mapstructure:"username" structs:"username"`
 	Password                 string      `json:"password" mapstructure:"password" structs:"password"`
+	AuthType                 string      `json:"auth_type" mapstructure:"auth_type" structs:"auth_type"`
+	Credentials              string      `json:"credentials" mapstructure:"credentials" structs:"credentials"`
 	DisableEscaping          bool        `json:"disable_escaping" mapstructure:"disable_escaping" structs:"disable_escaping"`
 
+	// cloud options here - cloudDriverName is globally unique, but only needs to be retained for the lifetime
+	// of driver registration, not from run to run.
```

Comment:
I think lifetime of driver registration means Config Create -> Config Delete. Is that correct? What does "from run to run" mean? Plugin request -> plugin request?

Reply:
By run to run, I mean like across a restart, or anything that "resets" non-persisted parts of the backend.